specs/protocol/deputy-pause-module.md

```diff
@@ -203,44 +183,44 @@ We assume that the Deputy key configured is actually capable of creating signatu
 
 ## System Invariants
 
-### iSCP-001: Superchain-wide pause can be activated within a short bounded time of authorization
+### iSCP-001: Pause can be activated within a short bounded time of authorization
 
-It is an important system-level invariant that the Superchain-wide pause functionality can be
-activated within a short, bounded amount of time of its authorization by the standard process that
-approves the usage of this pause.
+It is an important system-level invariant that the pause functionality can be activated within a
+short, bounded amount of time of its authorization by the standard process that approves the usage
+of this pause.
```

Comment:
Should we define a specific upper bound instead of just saying a "short" amount of time? Maybe 30 minutes? (really it can probably be even shorter since this is only considering post-authorization)

Reply:
30 minutes seems ok (arguably even long) if it's post-authorization (which was my assumption, given "iSCP-001: Pause can be activated within a short bounded time of authorization"), but agreed it should be longer if that time includes authorization